### PR TITLE
Annotorious upgrade, delete key fix (#1221), polygon tool (#1311)

### PR DESCRIPTION
**Pull request by blms**
## In this PR

- Fix a small issue with the enlarge button trying to initialize in the editor
- Upgrade annotorious, annotorious-toolbar, and annotorious-openseadragon to latest versions

Per #1221:
- Use the new `disableDeleteKey` option in Annotorious to fix a bug with editing annotation labels

Per #1311:
- Add a toolbar with the rectangle and polygon tools from Annotorious to allow creation of polygon image annotations within the editor

### CODE REVIEW

File: geniza/corpus/templates/corpus/snippets/document_transcription.html [hunk 1]
```diff
@@ -115,14 +115,28 @@
...
                                 {% if edit_mode %}
+                                    {# popout and close button #}
                                     <button class="primary popout-close-button" type="button" aria-label="Close"><i class="ph-x"></i></button>
                                     <button class="popout-button" type="button">
                                         <svg>
                                             <use xlink:href="{% static 'img/ui/desktop/all/pop-out.svg' %}#pop-out-icon" />
                                         </svg>
                                         <span class="sr-only">Pop out</span>
                                     </button>
+                                    <fieldset class="toolbar">
```
Comment (rlskoeser) [inline geniza/corpus/templates/corpus/snippets/document_transcription.html:126]: Why are these added to the geniza template instead of to tahqiq?

I don't object necessarily, just want to understand

File: geniza/corpus/templates/corpus/snippets/document_transcription.html [hunk 2]
```diff
@@ -115,14 +115,28 @@
...
+                                        {# rectangle tool #}
+                                        <label class="rect-tool active-tool">
+                                            <input type="radio" checked />
+                                            <span class="sr-only">Rectangle tool</span>
```
Comment (rlskoeser) [inline geniza/corpus/templates/corpus/snippets/document_transcription.html:130]: not sure how meaningful "rectangle tool" is; how about rephrasing to describe better what these do?

```suggestion
                                            <span class="sr-only">Select a rectangle</span>
```

File: geniza/corpus/templates/corpus/snippets/document_transcription.html (hunk 1 above)
Comment (blms) [reply geniza/corpus/templates/corpus/snippets/document_transcription.html:126]: I hadn't thought to do that! Was conceptualizing it as a Geniza feature so put it in Geniza, but I think it makes more sense to pass the toolbar element (in this case the fieldset) as a param to tahqiq, and instantiate the button elements there.

File: geniza/corpus/templates/corpus/snippets/document_transcription.html (hunk 2 above)
Comment (blms) [reply geniza/corpus/templates/corpus/snippets/document_transcription.html:130]: Will put this suggestion into the tahqiq update, thanks!

File: sitemedia/js/controllers/annotation_controller.js [hunk 3]
```diff
@@ -89,7 +89,10 @@ export default class extends Controller {
 
         // enable annotorious-tahqiq
         const { config, canvasURL, manifestId, annotationContainer } = settings;
-        const anno = Annotorious(viewer);
+        const anno = Annotorious(viewer, { disableDeleteKey: true });
+
+        // get toolbar element
+        const toolbarContainer = this.element.querySelector(".tahqiq-toolbar");
```
Comment (rlskoeser) [inline sitemedia/js/controllers/annotation_controller.js:95]: do you still need to bind this here? what does the annotation controller need to do with it?

Comment (blms) [reply sitemedia/js/controllers/annotation_controller.js]: The annotation controller needs to pass it to tahqiq instantiation, but doesn't need to define a variable for it so I've simplified.

**Review by rlskoeser — APPROVED** [review]
